packages/pricing/src/migrations/Migration20240312163523.ts

```diff
@@ -0,0 +1,18 @@
+import { Migration } from "@mikro-orm/migrations"
+
+export class Migration20240312163523 extends Migration {
+  async up(): Promise<void> {
```

Comment:
question: Is the new index missing here?

Reply:
no, its already present in the first migration. It just got missed in the model.